update core to vCPU

Drewm3 opened the pull request:
Updating to use the term vCPU instead of core where appropriate to match the rest of the platform.

azurecla commented:
Hi __@LuisCabrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (luisca). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rmca14 commented:
Checking for ability merge delayed -- waiting for it to complete.

TheBharath commented on articles/machine-learning/machine-learning-publish-web-service-to-azure-marketplace.md:
```diff
@@ -13,7 +13,7 @@
 	ms.devlang="na" 
 	ms.topic="article" 
-	ms.date="09/01/2015" 
+	ms.date="02/12/2016" 
```
+ms.date+"05/16/2016"

Drewm3 committed:
update core to vCPU

Updating to use the term vCPU instead of core where appropriate to match the rest of the platform.
